jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/ApplyServiceTest.java

```diff
@@ -362,6 +367,72 @@ public void testGetK8sListWithNamespaceFirstWithProject() {
         assertTrue(result.get(0) instanceof Project);
     }
 
+    @Test
+    public void testApplyToMultipleNamespaceNoNamespaceConfigured() throws InterruptedException {
+        // Given
+        ConfigMap configMap = new ConfigMapBuilder().withNewMetadata().withName("cm1").withNamespace("ns1").endMetadata().build();
+        Ingress ingress = new IngressBuilder().withNewMetadata().withName("ing1").withNamespace("ns2").endMetadata().build();
+        ServiceAccount serviceAccount = new ServiceAccountBuilder().withNewMetadata().withName("sa1").endMetadata().build();
+
+        List<HasMetadata> entities = new ArrayList<>();
+        entities.add(configMap);
+        entities.add(serviceAccount);
+        entities.add(ingress);
+        WebServerEventCollector collector = new WebServerEventCollector();
+        mockServer.expect().post().withPath("/api/v1/namespaces/ns1/configmaps")
+                .andReply(collector.record("configmap-ns1-create").andReturn(HTTP_OK, configMap))
+                .once();
+        mockServer.expect().post().withPath("/apis/networking.k8s.io/v1/namespaces/ns2/ingresses")
+                .andReply(collector.record("ingress-ns2-create").andReturn(HTTP_OK, ingress))
+                .once();
+        mockServer.expect().post().withPath("/api/v1/namespaces/default/serviceaccounts")
+                .andReply(collector.record("serviceaccount-default-create").andReturn(HTTP_OK, serviceAccount))
+                .once();
+        String configuredNamespace = applyService.getNamespace();
+        applyService.setNamespace(null);
+        applyService.setFallbackNamespace("default");
+
+        // When
+        applyService.applyEntities(null, entities, log, 5);
+
+        // Then
+        collector.assertEventsRecordedInOrder("configmap-ns1-create", "serviceaccount-default-create", "ingress-ns2-create");
+        assertEquals(6, mockServer.getMockServer().getRequestCount());
+        applyService.setFallbackNamespace(null);
+        applyService.setNamespace(configuredNamespace);
+    }
+
+    @Test
+    public void testApplyToMultipleNamespacesOverriddenWhenNamespaceConfigured() throws InterruptedException {
```

Comment:
I'm not sure what this test does compared to `testApplyToMultipleNamespaceNoNamespaceConfigured`